### PR TITLE
chore: Migrate Angular UI unit tests to React

### PR DESCRIPTION
**Pull request by igarashitm**
Fixes: #1195

### CODE REVIEW

File: ui/packages/atlasmap-standalone/src/App.test.tsx [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
 import ReactDOM from 'react-dom';
 import App from './App';
+import { AtlasmapProvider } from '@atlasmap/atlasmap';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
-});
+describe('AtlasMap standalone', () => {
```
Comment (igarashitm) [inline ui/packages/atlasmap-standalone/src/App.test.tsx:6]: Hi @riccardo-forina, I managed to get tests passing other than this one. This one fails with weird error `Target container is not a DOM element.`, do you see what's going on?

Comment (riccardo-forina) [reply ui/packages/atlasmap-standalone/src/App.test.tsx:6]: Hey great effort! For that check this out, we need to enrich the test document with a static doc with an ID that will be used by the app to inject the dialog:    https://github.com/syndesisio/syndesis/blob/c815aebc1931729a7377a15a3c35b75eb052372a/app/ui-react/packages/atlasmap-adapter/tests/DataMapperAdapter.spec.tsx#L46

Comment (igarashitm) [reply ui/packages/atlasmap-standalone/src/App.test.tsx:6]: thanks!

Comment (igarashitm) [reply ui/packages/atlasmap-standalone/src/App.test.tsx:7]: Turned out it was looking for a 'modals' div in underlying component. The standalone `App` is OK being empty.

**Comment by igarashitm** [comment]
CircleCI build fails, the `yarn test` process is terminated with SIGKILL. Possibly running out of memory.

**Comment by igarashitm** [comment]
OK... `--runInBand` solved exit 137 problem in UI atlasmap-core test.

**Pull request merged**